gsoft-inc/sg-orbit
# Replace cancellablePromise with a native AbortController

Issue: https://github.com/gsoft-inc/sg-orbit/issues/546

## What I did

- Replace our custom `cancellablePromise` utility by a native AbortController

## How to test

- Is this testable with Jest or Chromatic screenshots? No

- Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.
